app/src/conferences/urls.py

```diff
@@ -9,22 +9,22 @@
     re_path(r'^panel/$', views.admin_panel, name='admin'),
     re_path(r'^bus/$', views.bus_admin, name='bus_admin'),
     re_path(r'^bus/add/$', views.bus_add, name='bus_add'),
-    re_path(r'^bus/list/busbyuser$', views.list_csv_bus_by_user,
+    re_path(r'^bus/list/bus_by_user$', views.list_csv_bus_by_user,
             name='list_csv_bus_by_user'),
-    re_path(r'^bus/list/allusersbybus$', views.list_csv_all_users_by_bus,
+    re_path(r'^bus/list/all_users_by_bus$', views.list_csv_all_users_by_bus,
             name='list_csv_all_users_by_bus'),
-    re_path(r'^bus/list/paidusersbybus$', views.list_csv_paid_users_by_bus,
+    re_path(r'^bus/list/paid_users_by_bus$', views.list_csv_paid_users_by_bus,
             name='list_csv_paid_users_by_bus'),
     re_path(r'^bus/(?P<pk>\d+)/update/$', views.bus_add, name='bus_update'),
     re_path(r'^bus/(?P<pk>\d+)/people/$', views.bus_people, name='bus_people'),
```

Comment:
Should we rename `bus` to `transport` or we don't want to go that far? :thinking:

Reply:
This should go in the next PR where I'll change the whole model name to `Transport`.